contracts/SmartSession.sol

```diff
@@ -270,39 +273,35 @@ contract SmartSession is ISmartSession, SmartSessionBase, SmartSessionERC7739 {
         // action policies have to be checked
         if (selector == IERC7579Account.execute.selector) {
             // Decode ERC7579 execution mode
-            ExecutionMode mode = userOp.callData.get7579ExecutionMode();
-            CallType callType;
```

Comment:
I had to move this into a library to lose the stack allocation of mode. 
this avoids stack too deep